app/services/create_github_repo_service.rb

```diff
@@ -239,16 +228,37 @@ def repo_from_template_options
     }
   end
 
+  # rubocop:disable MethodLength
+  def handle_template_error(error, options)
+    raise Result::Error.new errors(:template_repository_not_found), error.message if error.class == GitHub::NotFound
```

Comment:
Not a blocker, but I wonder if it would be more clear to add this as a specific matching rescue at https://github.com/education/classroom/pull/2207/files#diff-63d3a0a1bbef5e8b4071f230c0a305aaR85. In there we could re-throw the Result::Error as we are doing and it would be a little closer to the concern. 🤷‍♂

Reply:
Makes sense! 👍